README.md

```diff
@@ -118,14 +118,22 @@ defmodule Myapp.Plugs.Authentication do
     body       = get_body(conn)
 
     %{
+      query_string: query_string,
       req_headers: req_headers,
       request_path: request_path,
       method: method,
     } = conn
 
+    full_path =
```

Comment:
for generating a path, should we use the `URI` module?

Reply:
@BlakeWilliams are you proposing something like `URI.encode(full_path)` after concatenating the string, or something else entirely?

Reply:
Sorry, I didn't see this comment before merging. I have another README update on top of this one (https://github.com/TheGnarCo/api_auth_ex/pull/38). Maybe we can move discussion about the example code and if/how it can be better there.

Reply:
I was thinking of creating a URI struct then using it to generate the path for us.

eg:

```
URI.parse(full_path) |> Map.put(:query, query_string) |> URI.to_string()
```

Might be overkill in this case, but it handles the conditional for us and maybe some other cases.